recipes/seamm-datastore/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "seamm-datastore" %}
+{% set version = "2021.11.21" %}
+{% set sha256 = "ac0af62629dba57e92f81861981eef9bf8557cdca00d5f443e4022c5e510aeb0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/seamm-datastore/seamm_datastore-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools
+  run:
```

Comment:
The package appears to do a poor job at specifying dependencies, but is `marshmallow_sqlalchemy` also required?

Reply:
Yes, along with several other packages as you noted. I believe the requirements are now correct and have tested installing from PyPi in a clean conda environment. At least all the tests work.